setup.py

```diff
@@ -23,6 +23,7 @@
     "pyarrow",
     "pyyaml>=5.1",
     "treelib",
+    "pydantic<2",
```

Comment:
Since this is an indirect dependency, could you add a comment on when it can be removed again?

Reply:
updated with comment